evennia/web/webclient/static/webclient/css/webclient.css

```diff
@@ -94,31 +94,54 @@ div {margin:0px;}
 
 /* Main scrolling message area */
 #messagewindow {
-    position: relative;
-    height: 90%;
+    position: absolute;
     overflow: auto;
     padding: 1em;
+    box-sizing: border-box;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 70px;
 }
 
 /* Input area containing input field and button */
 #inputform {
     position: absolute;
     width: 100%;
-    padding: .8em 0;
-    bottom: -50px;
+    padding: 0;
+    bottom: 0;
+    margin: 0;
+    padding-bottom: 10px;
+    border-top: 1px solid #555;
 }
 
 #inputcontrol {
-    padding: 0 .8em;
-    overflow: auto
+    width: 100%;
+    padding: 0;
 }
 
 /* Input field */
+#inputfield, #inputsend, #inputsizer {
+    display: block;
+    box-sizing: border-box;
```

Comment:
I know I looked at box-sizing for the original code but got the impression it was a relatively new css feature. Then again, it all comes down to where to put the lowest browser denominator, so I guess this is okay. I did get the impression that one needs vendor prefixes to handle medium-old browsers though, such as mentioned here: https://css-tricks.com/box-sizing/#article-header-id-7